unst_msh_gen/README.md

```diff
@@ -45,5 +45,27 @@ NOTE: The output mesh will have -180:180 longitude, you can convert this by unis
 	output_file_path: shifted mesh in gmsh format with 0:360 long
 
 
+6- Using variable mode:
```

Comment:
Please note that the markdown formatting is not quite as expected when you look at this document in github.  An update in formatting would improve readability in github.